kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -301,6 +301,43 @@ struct GetCategories {
   }
 };
 
+struct GetLatestCategoryInfo {
+  const bool read_only = true;
+  std::string description() const {
+    return "getLatestCategoryInfo CATEGORY-ID [BLOCK-VERSION]\n"
+           "  Returns the latest blockID that contains the givenb category, started from BLOCK-VERSION. If "
+           "BLOCK-VERSION is not set, we start from the lastReachableBlock";
+  }
+
+  std::string execute(const KeyValueBlockchain &adapter, const CommandArguments &args) const {
+    if (args.values.empty()) {
+      throw NotFoundException{"No Category ID was given"};
+    }
+    auto latestBlockID = adapter.getLastReachableBlockId();
+    if (args.values.size() == 2) {
```

Comment:
In all other cases we have >= for the number of arguments. Maybe it should be the same here?

Reply:
done